processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByRowProcessor.java

```diff
@@ -133,17 +127,15 @@ public boolean apply(Row input)
           @Override
           public CloseableGrouperIterator<RowBasedKey, Row> make()
           {
-            final List<Closeable> closeOnFailure = Lists.newArrayList();
+            final List<Closeable> closeOnExit = Lists.newArrayList();
```

Comment:
It seems that the order of closing matters. I don't want to change it in this PR.